.travis.yml

```diff
@@ -311,16 +209,6 @@ jobs:
         - pip install sphinx
       before_script: python create.py && python config.py
       script: make --directory docs doctest
-    - name: "Python 3.8, Django 2.1, SQLite - Doc Test"
```

Comment:
The test for "Python 3.8, Django 2.0, SQLite - Doc Test" should also be removed.